src/maintainer/updating_pkgs.rst

```diff
@@ -22,7 +22,7 @@ For updates, using a branch in the main repo is discouraged because,
    This means if you push a version update to a branch and then create a :term:`PR`, conda packages will be published to anaconda.org before the PR is merged.
 
 .. important::
-  For these reasons, maintainers are asked to fork the feedstock, push to a branch in the fork and then open a PR to the ``conda-forge`` repo.
+  For these reasons, maintainers are asked to fork the feedstock to their personal account, push to a branch in the fork and then open a PR to the ``conda-forge`` repo.
```

Comment:
I'm not sure about this wording. This seems to suggest using forks from orgs is not allowed, which Anaconda and a few others (including myself) do. While it's true that the re-rendering bot currently does not work in this case (GitHub disabled this checkbox recently), users are still free to re-render locally. We shouldn't block orgs from sending PRs from org repos.

Reply:
Hmh, what about:

---
For these reasons, maintainers are asked to fork the feedstock, push to a branch in the fork and then open a PR to the ``conda-forge`` repo.
**Please note:** The re-rendering bot does not work with forks from organisations. For best experience, please fork to your personal account.